tests/test_interpreter.py

```diff
@@ -167,3 +205,27 @@ def test_find_python_interpreter_missing_on_path_raises(monkeypatch):
         find_python_interpreter(interpreter)
         assert "Python Launcher" in str(e)
         assert "on your PATH" in str(e)
+
+
+@pytest.mark.skipif(sys.platform.startswith("win"), reason="Looks for python3")
```

Comment:
I think we could deduplicate the tests and simply add a parametrisation for the executable names, then test for the platform within the test and present an according `assert` statement?